app/components/Receive/ReceivePanel/ReceivePanel.jsx

```diff
@@ -14,26 +18,48 @@ type Props = {
   address: string
 }
 
-export default class ReceivePanel extends React.Component<Props> {
-  canvas: ?Node
+type State = {
+  asset: ?string,
+  amount: ?number
+}
 
-  componentDidMount () {
-    QRCode.toCanvas(this.canvas, this.props.address, { version: 5 }, (err) => {
-      if (err) console.error(err)
-    })
+export default class ReceivePanel extends React.Component<Props, State> {
+  image: ?HTMLImageElement
+
+  state = {
+    asset: null,
+    amount: 0
   }
 
   render () {
     const { className, address } = this.props
+    const { asset, amount } = this.state
+    const symbols = ['-', 'NEO', 'GAS', ...Object.keys(TOKENS)]
 
     return (
       <Panel className={classNames(styles.receivePanel, className)} renderHeader={this.renderHeader}>
         <div className={styles.qrcode}>
-          <canvas ref={this.registerRef} />
+          <img ref={this.registerRef} />
```

Comment:
I think its ok to do it inline `ref={(el: HTMLImageElement) => { this.image = el })`

Reply:
updated